src/EnrollU2FController.js

```diff
@@ -63,7 +65,13 @@ function (Okta, FormType, FormController, Footer, Q) {
           u2f.register(appId, registerRequests, [], function (data) {
             self.trigger('errors:clear');
             if (data.errorCode && data.errorCode !== 0) {
-              deferred.reject({ responseJSON: {errorSummary: 'Error Code: ' + data.errorCode}});
+              if (data.errorCode === 5) { // TIMEOUT
+                self.trigger('abort');
+                deferred.resolve(transaction);
+              }
+              else {
+                deferred.reject({xhr: {responseJSON: {errorSummary: 'Error Code: ' + data.errorCode}}});
```

Comment:
this should be changed to something like
```
var errorMessage;
switch(data.errorCode) {
  case 1: errorMessage = Okta.loc('u2f.error.unknown');
...
responseJson: { errorSummary: errorMessage + '(' + data.errorCode + ')';
```
In that way we have the errors localized, but still we can see the error code at the end (it might happen that we have the same error message for codes 2-3, so we still need to identify them somehow)